project/AppDependencies.scala

```diff
@@ -25,7 +25,7 @@ object AppDependencies {
       ExclusionRule("com.fasterxml.jackson.dataformat", "jackson-dataformat-yaml"),
       ExclusionRule("com.fasterxml.jackson.datatype", "jackson-datatype-jsr310")
     ),
-    "org.scala-lang.modules"            % "scala-xml_2.13"                  % "1.3.0",
+//    "org.scala-lang.modules"            % "scala-xml_2.13"                  % "1.3.0",
```

Comment:
Does this need removing?

Reply:
It needs removing and replacing with that new "pick the version based on scala" construct

Reply:
line deleted